openeo_udp/eurac_pv_farm_detection/udf_eurac_pvfarm_onnx.py

```diff
@@ -0,0 +1,189 @@
+import functools
+import logging
+import os
+import sys
+import zipfile
+from typing import Dict
+
+import numpy as np
+import requests
+import xarray as xr
+
+
+def _setup_logging():
+    logging.basicConfig(level=logging.INFO)
+    logger = logging.getLogger(__name__)
+    return logger
```

Comment:
Instead of using `logger`, I would promote usage of `inspect` (https://open-eo.github.io/openeo-python-client/udf.html#udf-logging-with-inspect)